packages/cli/lib/index.ts

```diff
@@ -4,25 +4,27 @@
  * Copyright (c) 2024 Nango, all rights reserved.
  */
 
-import { Command } from 'commander';
 import fs from 'fs';
-import chalk from 'chalk';
-import figlet from 'figlet';
 import path from 'path';
+
+import { nangoConfigFile } from '@nangohq/nango-yaml';
```

Comment:
I am not sure about the empty line. I'd prefer without it but I won't fight for it

Reply:
you mean no empty lines between groups or for this group in particular?

Reply:
I mean no empty lines between groups